source/usage-examples/updateMany.txt

```diff
@@ -0,0 +1,101 @@
+.. _csharp-update-many:
+
+=====================
+Update Many Documents
+=====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can update more than one document at once using the `UpdateMany() <{+api-root+}/M_MongoDB_Driver_IMongoCollection_1_UpdateMany.htm>`__ method on
```

Comment:
Good point; that confusion never occurred to me. I intentionally added "at once" because it's also true that you can update multiple documents using `UpdateOne()` (you just need to call it multiple times, obviously). Is there a better way to convey that you can do multiple updates with a single method call? 

* You can update multiple documents in a single operation...
* You can update multiple documents with a single method call...
* Etc.

Reply:
I think "you can update multiple documents using the..." is sufficient, but "you can update multiple documents with one operation using the" is also alright.